elixir/apps/web/test/web/controllers/auth_controller_test.exs

```diff
@@ -258,8 +258,6 @@ defmodule Web.AuthControllerTest do
 
       assert redirected_to = redirected_to(conn)
       assert redirected_to_uri = URI.parse(redirected_to)
-      assert redirected_to_uri.scheme == "https"
-      assert redirected_to_uri.host == "app.firez.one"
```

Comment:
We are missing tests for both of behaviours to make sure we do a full redirect on apple and path redirect on android, other than that LGTM.

Reply:
Good catch. Will add these.

Reply:
Added!